driver-core/src/main/com/mongodb/operation/AggregateExplainOperation.java

```diff
@@ -153,7 +159,7 @@ public BsonDocument execute(final ReadBinding binding) {
             @Override
             public BsonDocument call(final Connection connection) {
                 validateCollation(connection, collation);
-                return executeWrappedCommandProtocol(binding, namespace.getDatabaseName(), getCommand(), connection);
+                return CommandOperationHelper.executeCommand(binding, retryReads, namespace.getDatabaseName(), getCommand(), connection);
```

Comment:
Nit: make retryReads the last parameter since it's kind of the optional one.  (You can use IntelliJ's Change Signature auto-refactoring to make this change)

Reply:
Done.